src/detect-engine-alert.c

```diff
@@ -34,6 +34,13 @@ static Signature g_tag_signature;
 /** tag packet alert structure for tag alerts */
 static PacketAlert g_tag_pa;
 
+/* For WARN_ONCE, a record of warnings that have already been
+ * issued. */
+static thread_local bool once_err[SC_ERR_MAX];
```

Comment:
I changed the name, as I wasn't sure if that could cause issues or not.

Reply:
> @jlucovsky any thoughts on how this works wrt multiple static declarations with the same name in different modules?

Because it's `static`, the scope should be local and treated like any other `static` with the same name. 

A quick experiment confirms that the scope is restricted to the current source module.